compiler/damlc/tests/daml-test-files/ExceptionSyntax.daml

```diff
@@ -0,0 +1,36 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF 1.dev
+-- @QUERY-LF [ .modules[].exceptions[] ] | length == 1
+-- @WARN Modules compiled with the DatatypeContexts language extension
+{-# LANGUAGE DatatypeContexts #-}
```

Comment:
I’m confused, why do we need to turn this on? Do our users also have to turn this on?

Reply:
Oh, oops, that's just a leftover from copying the other module. I'll remove it!